### PR TITLE
Added Getting Started guide

### PR DESCRIPTION
**Pull request by ghost**
Based on #495

### CODE REVIEW

**Review by fhemberger — DISMISSED** [review]

**Comment by amandeepmittal** [comment]
Should not it require the `http` module using ES6 syntax (using `const` instead of `var`)?

**Comment by vsemozhetbyt** [comment]
FWIW, if we lint this according to the main repository rules:
```console
  12:1   error  Unexpected var, use let or const instead          no-var
  16:1   error  Expected indentation of 2 spaces but found 1 tab  indent
  16:2   error  Unexpected tab character                          no-tabs
  16:21  error  A space is required after '{'                     object-curly-spacing
  16:49  error  A space is required before '}'                    object-curly-spacing
  17:1   error  Expected indentation of 2 spaces but found 1 tab  indent
  17:2   error  Unexpected tab character                          no-tabs
```

**Comment by saadq** [comment]
I agree with @amandeepmittal . For consistency's sake, it would probably be a good idea to use the same exact code as https://nodejs.org/en/about/

```js
const http = require('http');

const hostname = '127.0.0.1';
const port = 3000;

const server = http.createServer((req, res) => {
  res.statusCode = 200;
  res.setHeader('Content-Type', 'text/plain');
  res.end('Hello World\n');
});

server.listen(port, hostname, () => {
  console.log(`Server running at http://${hostname}:${port}/`);
});
```

File: locale/en/docs/guides/getting-started-guide.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  res.setHeader('Content-Type', 'text/plain');
+  res.end('Hello World\n');
+});
+
+server.listen(port, hostname, () => {
+  console.log(`Server running at http://${hostname}:${port}/`);
+});
+```
+
+After that, run your web server using ``` node app.js ```, visit http://localhost:8080, and you will see a message 'Hello World!'
```
Comment (vsemozhetbyt) [inline locale/en/docs/guides/getting-started-guide.md]: This line needs to be updated: `8080` -> `3000` and `'Hello World!'` -> `'Hello World'`.

**Review by vsemozhetbyt — APPROVED** [review]

**Comment by fhemberger** [comment]
@jevtovich Awesome, thank you!

**Comment by ghost** [comment]
@fhemberger that you as well, and I'm sorry I made some mistakes I'm new to git :smile:

**Comment by fhemberger** [comment]
@jevtovich No worries, happens to everybody all the time. 😀

**Pull request merged**